back/tests/cmd/openbsrv-integration/main.go

```diff
@@ -0,0 +1,324 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/OpenEugene/openboard/back/internal/pb"
+	"google.golang.org/grpc"
+)
+
+func main() {
+	conn, err := grpc.Dial(":4242", grpc.WithInsecure())
+	if err != nil {
+		fmt.Println(err)
+	}
+	defer conn.Close()
+
+	userClnt := pb.NewUserSvcClient(conn)
+	fmt.Println("=====================Start User Service Tests=====================")
+	userSvcAddRoles(conn, userClnt)
+	userSvcAddUsers(conn, userClnt)
+	userSvcFndUsers(conn, userClnt)
+	userAID := userSvcFndUserA(conn, userClnt)
+	userSvcDelUserA(conn, userClnt, userAID)
+	_ = userSvcFndUserA(conn, userClnt)
+
+	fmt.Println("=====================Start Post Service Tests=====================")
```

Comment:
I think it would be better to use subtests for this sort of need (i.e. namespacing tests). We should talk it over, though.

Reply:
I agree--this week, after I wrote these tests, I read some other people's test and they're much better organized. I would like to do something that's neater and easier to follow.